plugin/al/lib/AL_USDMaya/AL/usdmaya/PluginRegister.h

```diff
@@ -51,7 +51,7 @@
 #include "pxr/base/plug/plugin.h"
 #include "pxr/base/plug/registry.h"
 
-#if (PXR_MAJOR_VERSION > 0) || (PXR_MINOR_VERSION >= 19 && PXR_PATCH_VERSION >= 5) 
+#if USD_VERSION_NUM >= 1903
```

Comment:
Should the version number be 1905 here, based on the incantation being replaced?